tensorboard/webapp/feature_flag/store/feature_flag_metadata.ts

```diff
@@ -110,11 +110,6 @@ export const FeatureFlagMetadataMap: FeatureFlagMetadataMapType<FeatureFlags> =
       queryParamOverride: 'enableScalarColumnCustomization',
       parseValue: parseBoolean,
     },
-    enableScalarColumnContextMenus: {
-      defaultValue: false,
```

Comment:
The preferred way to do this would have been to set defaultValue: true.

That way rolling it back is very slightly more straightforward - greater chance of avoiding merge conflicts and having a clean rollback commit.

And then delete the code, say, one or two weeks later?

But I leave that up to you to decide which way to go.

Reply:
Whoops sorry I see now that your request in the email was just to set the defaultValue.

I'll send another PR for the defaultValue update and we can just apply this PR later when everything looks good.